README.md

```diff
@@ -183,6 +183,68 @@ resource "ocm_cluster_wait" "rosa_cluster" {
 ￼}
```

Comment:
please remove the ￼

Reply:
@bardielle  can you elaborate?

Reply:
In your lase commit you added the line 181 and 183 with a sign that looks different:  ￼
`￼  cluster = ocm_cluster.rosa_cluster.id
   timeout = 30 
￼}`

Reply:
hmm. I do not see that in either diff/file difference on github 🤔